pkg/scheduler/api/constants.go

```diff
@@ -0,0 +1,6 @@
+package api
+
+const (
+	// EnablePredicateErrCacheKey is the key whether predicate error cache is enabled
+	EnablePredicateErrCacheKey = "enable.predicateErrorCache"
```

Comment:
Can we refer to plugin's option definition？Because it's also an option or param of action.

Reply:
such as `allocate.predicateErrorCacheEnable` or `predicateErrorCacheEnable`?

Reply:
Yeah.

Reply:
Has changed to `predicateErrorCacheEnable`.